### DISCUSSION TITLE
Why can't I use Korean in the title?

### DISCUSSION THREAD

**Discussion by mbc21c** [alagrede/znote-app#25] [General]
It doesn't work when I try to write the title in Korean.

**Comment by alagrede** [comment]
Hello @mbc21c,
It's not a specific reason I'm confused I never try to use Korean in the app 😅
I will try to change this for next releases.
Cheers,

**Comment by alagrede** [comment]
@mbc21c Work in progress. Coming soon! 😁 
<img width="1499" alt="korean-test" src="https://user-images.githubusercontent.com/5312754/122370082-0fde4600-cf5f-11eb-8aef-b2bdb2dd2e12.png">

**Comment by alagrede** [comment]
Made! It is now available in version 0.7.7 😎 
Thank you for your report.

**Reply by alagrede** [reply]
@mbc21c should be good now.
